### PR TITLE
fix empty sliders bug

### PR DESCRIPTION
**Pull request by mixcas**
js was breaking if the sliders where empty

### CODE REVIEW

File: single.php [hunk 1]
```diff
@@ -13,9 +13,9 @@
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
-      $meta = get_post_meta($post->ID);
-      $photos = get_post_meta($post->ID, '_igv_photos');
-      $drawings = get_post_meta($post->ID, '_igv_drawings');
+    $meta = get_post_meta($post->ID);
+    $photos = get_post_meta($post->ID, '_igv_photos');
+    $drawings = get_post_meta($post->ID, '_igv_drawings');
```
Comment (varffs) [inline single.php:18]: 2 extra calls to get meta when meta is already in the `$meta` variable? I think this can all just go from that no?

Comment (varffs) [inline single.php:18]: But not sure if it is that weird array parse from database thing?

Comment (mixcas) [inline single.php:18]: not my code _shrugs_
fix is not about it

Comment (mixcas) [inline single.php:18]: i thought this was already reviewed…

Comment (varffs) [inline single.php:18]: _shrugs_

**Pull request merged**